test/test_browser.py

```diff
@@ -5495,17 +5495,21 @@ def test_audio_worklet_post_function(self, args):
   def test_audio_worklet_modularize(self, args):
     self.btest_exit('webaudio/audioworklet.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMODULARIZE=1', '-sEXPORT_NAME=MyModule', '--shell-file', test_file('shell_that_launches_modularize.html')] + args)
 
-  # Tests multiple inputs, forcing a larger stack (note: passing BROWSER_TEST is
-  # specific to this test to allow it to exit rather than play forever).
+  # Tests an AudioWorklet with multiple stereo inputs mixing in the processor
+  # via a varying parameter to a single stereo output (touching all of the API
+  # copying from structs)
   @parameterized({
     '': ([],),
     'minimal_with_closure': (['-sMINIMAL_RUNTIME', '--closure=1', '-Oz'],),
   })
-  def test_audio_worklet_stereo_io(self, args):
+  @no_wasm64('https://github.com/emscripten-core/emscripten/pull/23508')
+  @no_2gb('https://github.com/emscripten-core/emscripten/pull/23508')
+  @requires_sound_hardware
+  def test_audio_worklet_params_mixing(self, args):
     os.mkdir('audio_files')
     shutil.copy(test_file('webaudio/audio_files/emscripten-beat.mp3'), 'audio_files/')
     shutil.copy(test_file('webaudio/audio_files/emscripten-bass.mp3'), 'audio_files/')
-    self.btest_exit('webaudio/audioworklet_in_out_stereo.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-DBROWSER_TEST'] + args)
+    self.btest_exit('webaudio/audioworklet_params_mixing.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-DTEST_AND_EXIT'] + args)
```

Comment:
You use `-DTEST_AND_EXIT=1` in some places just `-DTEST_AND_EXIT` here.   If the later is enough, perhaps just use that everywhere?

Reply:
Without the parameter one second, with it's 10 otherwise the interactive test is too short to interact.

Maybe I change to `1` for a second, `10` for ten seconds?

Reply:
Maybe just make the interactive tests run forever?  Or until some button is clicked.   I'm not really sure that the overriding pattern is for them.   But whatever it is perhaps you don't need to defined `TEST_AND_EXIT` at all?

Reply:
BTW, do you ever run the interactive tests more than one at a time (i.e. a sequence of them?).  I never got into the habit of doing so and I imagine many of them don't exit correctly.. but maybe thats fine.

Reply:
I've never run them unattended, since some will just time out and fail waiting, and I only run the ones related to what I'm looking at.

I'll remove `TEST_AND_EXIT` for interactive, that simplifies things more. It'll be tomorrow now.